psi4/src/psi4/fisapt/fisapt.cc

```diff
@@ -3670,16 +3898,16 @@ void FISAPT::find() {
 
     // ==> Uncoupled Targets <== //
 
-    auto Ind20u_AB_terms = std::make_shared<Matrix>("Ind20 [A<-B] (a x B)", na, nB + nb);
-    auto Ind20u_BA_terms = std::make_shared<Matrix>("Ind20 [B<-A] (A x b)", nA + na, nb);
+    auto Ind20u_AB_terms = std::make_shared<Matrix>("Ind20 [A<-B] (a x B)", na, nB + nb + 1); // add one entry for external potential 
```

Comment:
The definition concern is throughout. I'll stop bringing it up, at this point.

Reply:
See above.